src/app/asset-page/artstor-viewer/artstor-viewer.component.html

```diff
@@ -1,5 +1,37 @@
 <div class="asset-viewer" [class.fullscreen]="isFullscreen" [class.multi-view]="isMultiView" (contextmenu)="disableContextMenu($event)">
-    <!-- Large thumbnail fallback
+  <!--Viewer Buttons-->
```

Comment:
This block of html was practically moved as-is from the bottom, up here to the top. This ensures the viewer buttons (which are overlaid on top of the viewer) come BEFORE the viewer in terms of tab order.

There is one exception however and that is the `fullscreen-metadata` div (still at the bottom). We had to extract that div and leave it where it was, because we want that tab order to come AFTER the viewer. That html element still remains as-is too.